### PR TITLE
Added listTopics() in KakfaConsumer and queue-id support in deleteTopics() and describeTopics() in KafkaAdminClient

### PR DESCRIPTION
**Pull request by s-ourabh**

### CODE REVIEW

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopicsById.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+public class OkafkaDescribeTopicsById {
+	@Test
+	public void AdminTest() {
+        try (Admin admin = AdminClient.create(OkafkaSetup.setup())) {
+        	CreateTopicsResult result = admin.createTopics(Arrays.asList(
+					new NewTopic("TOPIC",5, (short)1)));
+        	Uuid createdTopicId = result.topicId("TOPIC").get();
+        	admin.close();
+        	
+        	Admin admin1 = AdminClient.create(OkafkaSetup.setup());
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopicsById.java:27]: why you have to take new admin client ? 
Also admin1 is not closed .

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopics.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import org.apache.kafka.clients.admin.TopicDescription;
 import org.apache.kafka.common.KafkaFuture;
 import org.apache.kafka.common.TopicCollection;
+import org.apache.kafka.common.Uuid;
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopics.java:13]: Do we use Uuid in this test ?

File: clients/src/test/java/org/oracle/okafka/tests/SimpleOkafkaAdmin.java [hunk 3]
```diff
@@ -16,9 +14,10 @@ public class SimpleOkafkaAdmin {
 
 	@Test
 	public void AdminTest() {
+		
         try (Admin admin = AdminClient.create(OkafkaSetup.setup())) {
 			CreateTopicsResult result = admin.createTopics(Arrays.asList(
-					new NewTopic("TEQ",5, (short)1)));
+					new NewTopic("topic",5, (short)1)));
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/SimpleOkafkaAdmin.java:20]: Why topicname  change?

File: clients/src/test/java/org/oracle/okafka/tests/SimpleOkafkaProducer.java [hunk 4]
```diff
@@ -26,7 +26,7 @@ public void ProducerTest() {
 			Future<RecordMetadata> lastFuture = null;
 			int msgCnt = 1000;
 			for(int i=0;i<msgCnt;i++) {
-				ProducerRecord<String, String> producerRecord = new ProducerRecord<String, String>("TEQ", i+"", "Test message # " + i);
+				ProducerRecord<String, String> producerRecord = new ProducerRecord<String, String>("TE1", i+"", "Test message # " + i);
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/SimpleOkafkaProducer.java:29]: Why topicname change?

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDeleteTopicById.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	@Test
+	public void DeleteTopicByIdTest() {
+		 try (Admin admin = AdminClient.create(OkafkaSetup.setup())) {
+			 
+			CreateTopicsResult result = admin.createTopics(Arrays.asList(
+						new NewTopic("TOPIC",5, (short)1)));
+			Uuid createdTopicId = result.topicId("TOPIC").get();
+			admin.close();
+			
+			Admin admin1 = AdminClient.create(OkafkaSetup.setup());
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/OkafkaDeleteTopicById.java:27]: admin1 is not closed.

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDeleteTopic.java [hunk 6]
```diff
@@ -14,8 +16,9 @@ public class OkafkaDeleteTopic {
 	
 	@Test
 	public void DeleteTopicTest() {
+
 		try (Admin admin = AdminClient.create(OkafkaSetup.setup())) {
-			DeleteTopicsResult delResult = admin.deleteTopics(TopicCollection.TopicNameCollection.ofTopicNames(new ArrayList<String> (Arrays.asList("TEQ"))));
+			DeleteTopicsResult delResult = admin.deleteTopics(TopicCollection.TopicNameCollection.ofTopicNames(new ArrayList<String> (Arrays.asList("TOPIC"))));
```
Comment (ichokshi2109) [inline clients/src/test/java/org/oracle/okafka/tests/OkafkaDeleteTopic.java:21]: I need to understand the flow of test execution.  Make sure all tests run fine.

File: clients/src/main/java/org/oracle/okafka/common/utils/CreateTopics.java [hunk 7]
```diff
@@ -93,6 +95,12 @@ public static Map<String, Exception> createTopics(Connection jdbcConn, Map<Strin
 					 cStmt.setInt(2, details.numPartitions);
 					 cStmt.setLong(3,retentionSec);
 					 cStmt.execute();
+					 try {
+						 topicIdMap.put(topic, AQClient.getIdByTopic(jdbcConn,topic));
+					 } catch(Exception e){
+						 topicIdMap.put(topic, Uuid.ZERO_UUID);
```
Comment (ichokshi2109) [inline clients/src/main/java/org/oracle/okafka/common/utils/CreateTopics.java:101]: We need to log the exception

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopics.java (hunk 2 above)
Comment (s-ourabh) [reply clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopics.java:13]: No, i will remove that.

File: clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopicsById.java (hunk 1 above)
Comment (s-ourabh) [reply clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopicsById.java:27]: yes, i forgot to discuss this with you, actually there was some issue while using the  same admin client instance.

**Pull request closed without merging**